modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/data_set_display/management_bar/components/FiltersDropdown.js

```diff
@@ -83,20 +75,51 @@ function FiltersDropdown() {
 				</button>
 			}
 		>
-			<ClayDropDown.Search
-				onChange={(event) => setQuery(event.target.value)}
-				value={query}
-			/>
-			{visibleFilters.length ? (
-				<ClayDropDown.ItemList>
-					{visibleFilters.map((item) => (
-						<DropdownFilterItem key={item.id} {...item} />
-					))}
-				</ClayDropDown.ItemList>
+			{activeFilterId ? (
+				<>
+					<li className="dropdown-subheader">
+						<ClayButtonWithIcon
+							className="btn-filter-navigation"
+							displayType="unstyled"
+							onClick={() => setActiveFilterId(null)}
+							small
+							symbol="angle-left"
+						/>
+						{activeFilter.label}
+					</li>
+					<Filter {...{...activeFilter, actions}} />
+				</>
 			) : (
-				<div className="dropdown-section text-muted">
-					{Liferay.Language.get('no-filters-were-found')}
-				</div>
+				<>
+					<li className="dropdown-subheader">
+						{Liferay.Language.get('filters')}
+					</li>
+					<ClayDropDown.Search
+						onChange={(e) => setQuery(e.target.value)}
+						value={query}
+					/>
+					<ClayDropDown.Divider className="m-0" />
+					{visibleFilters.length ? (
+						<ClayDropDown.ItemList>
+							{visibleFilters.map((item) => (
+								<ClayDropDown.Item
+									active={
+										item.value !== undefined &&
+										item.value !== null
+									}
+									key={item.id}
+									onClick={() => setActiveFilterId(item.id)}
+								>
+									{item.label}
+								</ClayDropDown.Item>
+							))}
+						</ClayDropDown.ItemList>
+					) : (
+						<div className="dropdown-section text-muted">
```

Comment:
Is this a `dropdown-section`? Don't know exactly what we're trying to accomplish here 😅